Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -0,0 +1,295 @@
+        func drawBuffers() {
+            // Ensure that there points to draw
+            guard !bufferPoints.isEmpty else {
+                status = .noPoints
+                return
+            }
+            
+            // Clear existing buffers graphics before drawing.
+            bufferGraphicsOverlay.removeAllGraphics()
+            tappedPointsGraphicsOverlay.removeAllGraphics()
+            
+            // Reduce the bufferPoints tuples into points and radii arrays.
+            let (points, radii) = bufferPoints.reduce(into: ([Point](), [Double]())) { (result, pointAndRadius) in
+                result.0.append(pointAndRadius.point)
+                result.1.append(pointAndRadius.radius)
+            }
+            
+            // Create the buffers.
+            // Notice: the radius distances has the same unit of the map's spatial reference's unit.
+            // In this case, the statePlaneNorthCentralTexas spatial reference uses US feet.
+            let bufferPolygon = GeometryEngine.buffer(around: points, distances: radii, shouldUnion: shouldUnion)
+            
+            // Add the tapped points to the tappedPointsGraphicsOverlay.
+            points.forEach { point in
+                tappedPointsGraphicsOverlay.addGraphic(Graphic(geometry: point))
+            }
+            
+            // Add the buffers to the bufferGraphicsOverlay.
+            bufferPolygon.forEach { buffer in
+                bufferGraphicsOverlay.addGraphic(Graphic(geometry: buffer))
+            }
+        }
+        
+        /// Add a point with its radius to the bufferPoints Array.
+        func addBufferPoint() {
+            // Check to ensure the tappedPoint is within the boundary
+            guard isWithinBounds(tappedPoint) else {
+                status = .outOfBoundsTap
+                return
+            }
+            
+            // Update the buffer radius with the text value.
+            let radiusInMiles = Measurement(value: Double(radiusInput)!, unit: UnitLength.miles)
+            
+            // The spatial reference in this sample uses US feet as its unit.
+            let radiusInFeet = radiusInMiles.converted(to: .feet).value
+            
+            // Add point with radius to bufferPoints Array.
+            bufferPoints.append((point: tappedPoint!, radius: radiusInFeet))
+        }
+        
+        /// Clear the bufferPoints and related graphics.
+        func clearBufferPoints() {
+            bufferPoints.removeAll()
+            bufferGraphicsOverlay.removeAllGraphics()
+            tappedPointsGraphicsOverlay.removeAllGraphics()
+        }
+    }
+}
+
+private extension CreateBuffersAroundPointsView {
+    /// An Enum for the sample status.
+    enum Status {
+        case addPoints
+        case bufferCreated
+        case outOfBoundsTap
+        case noPoints
+    }
+    
+    /// Get the text message associated with the current status for the overlay.
+    /// - Returns: A 'String' message.
```

Comment:
```suggestion
    /// - Returns: A status message.
```
Also we use single ticks for symbol names, so they look like code in the quick help.
![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/ae8b2950-a4f2-4d23-8b7d-29e24648fadd)